static/js/buddy_data.js

```diff
@@ -48,6 +41,22 @@ exports.get_user_circle_class = function (user_id) {
     }
 };
 
+exports.get_user_circle_status = function (user_id) {
```

Comment:
I would call this function something more like `status_description`.  I think having `user_circle` in the name is confusing, since the names here are pretty independent of whether our icon is a circle, square, or some other thing.

Reply:
Changed the name to `status_description`